.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+_A brief description of the pull request. Try to keep it non-technical. Please mark your pull request with the appropriate label(s)_
```

Comment:
I think we should have a list of which labels we should use and what's their definition (That is, what is considered a bug? What is considered an enhancement? Is it different than a new feature?).

After we have that, we should probably remove any labels we won't be using.